apps/chat/src/components/Chatbar/ChatFolders.tsx

```diff
@@ -213,14 +213,14 @@ export const ChatSection = ({
   );
 
   const rootFolders = useMemo(
-    () => folders.filter(({ folderId }) => !folderId),
+    () => folders.filter(({ folderId }) => folderId.split('/').length === 2),
```

Comment:
Create in `ConversationService` \ `PromptService` \ `FileService` as static method/property `isRootPath` because `.split('/').length === 2` is just magic code

Reply:
Simplified to just util method - in refactoring phase we need to recheck this to find something easier